src/pmacFilterControl.cpp

```diff
@@ -16,7 +16,6 @@
 const int MAX_ATTENUATION = 15;  // All filters in: 1 + 2 + 4 + 8
 const long POLL_TIMEOUT = 100;  // Length of ZMQ poll in milliseconds
 const int FILTER_COUNT = 4;  // Number of filters
-const int CONTINUOUS_MODE_TIMEOUT = 3;  // Seconds of no messages before setting max attenuation in continuous mode
```

Comment:
Squash Removed old timeout variable and Added timeout to status dict into the first commit